Homework 1 completed

daxdue opened the pull request.

Commit:
Homework 1 done.

komyagin commented on src/test/java/ru/training/at/hw1/DataProviders.java:
```diff
@@ -0,0 +1,13 @@
+package ru.training.at.hw1;
+
+public interface DataProviders {
+    String SUM_LONG_DATA_PROVIDER = "sum_long_data_provider";
+    String SUM_DOUBLE_DATA_PROVER = "sum_double_data_provider";
+    String SUB_LONG_DATA_PROVIDER = "sub_long_data_provider";
+    String SUB_DOUBLE_DATA_PROVIDER = "sub_double_data_provider";
+    String MUL_LONG_DATA_PROVIDER = "mul_long_data_provider";
+    String MUL_DOUBLE_DATA_PROVIDER = "mul_double_data_provider";
+    String DIV_LONG_DATA_PROVIDER = "div_long_data_provider";
+    String DIV_DOUBLE_DATA_PROVIDER = "div_double_data_provider";
+    String DIV_BY_ZERO_DATA_PROVIDER = "div_by_zero_data_provider";
```
I see the names of the constants, but don't see appropriate keywords for a declaration.

daxdue replied:
Fixed.

komyagin commented on src/test/java/ru/training/at/hw1/Tags.java:
```diff
@@ -0,0 +1,6 @@
+package ru.training.at.hw1;
+
+public interface Tags {
+    String ADD_SUB = "addSub";
+    String MULT_DIV = "multDiv";
```
I see the names of the constants, but don't see appropriate keywords for a declaration.

daxdue replied:
Fixed.

komyagin commented on src/test/java/ru/training/at/hw1/TestsDataProvider.java:
```diff
@@ -0,0 +1,92 @@
+        return new Object[][] {{0, 0}, {454, 0}, {-1767, 0},
+            {Long.MAX_VALUE, 0}, {Long.MIN_VALUE, 0}};
+    }
+
+
+
```
Why do you need so many blank lines?

daxdue replied:
Fixed.

komyagin commented on src/test/java/ru/training/at/hw1/DataProviders.java:
```diff
@@ -0,0 +1,13 @@
+package ru.training.at.hw1;
+
+public interface DataProviders {
```
Why do you use an interface for storing constants?

komyagin commented on src/test/java/ru/training/at/hw1/Tags.java:
```diff
@@ -0,0 +1,6 @@
+package ru.training.at.hw1;
+
+public interface Tags {
```
Why do you use an interface for storing constants?

komyagin commented on pom.xml:
```diff
@@ -4,9 +4,109 @@
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>3.0.0-M5</version>
```
If you use properties for testng and others why don't you use it for this?

komyagin commented on pom.xml:
```diff
@@ -4,9 +4,109 @@
+        <dependency>
+            <groupId>com.epam</groupId>
+            <artifactId>calculator</artifactId>
+            <version>1.0</version>
```
If you use properties for testng and others why don't you use it for this?

komyagin commented on pom.xml:
```diff
@@ -4,9 +4,109 @@
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-checkstyle-plugin</artifactId>
+                <version>3.1.2</version>
```
If you use properties for testng and others why don't you use it for this?

Commit:
Some issues fixed:
- Data provider's constants moved to TestsDataProvider class
- Added required modifiers for constants
- Removed unnecessary blank lines
- Added version properties for some dependencies in pom.xml

Commit:
Added calculator jar file, fixed calculator dependency definition in pom
